src/elements/common/sub-header/__tests__/SortButton.test.js

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+import { SortButtonBase } from '../SortButton';
+import Button from '../../../../components/button';
+import IconSort from '../../../../icons/general/IconSort';
+
+const intlMock = {
+    formatMessage: jest.fn().mockReturnValue('Sort'),
+};
```

Comment:
not sure if needed if this is global: https://github.com/box/box-ui-elements/blob/master/scripts/jest/mocks/lib-intl-mock.js

Reply:
I don't think this is global, both VSCode and `yarn test /path/to/test` throw errors - I could import it but I'd rather just use a custom mock